.github/workflows/ci.yml

```diff
@@ -60,12 +60,18 @@ jobs:
           - image: haskell:9.8.4
             os: ubuntu-24.04-arm  # Exact version is not important, as it's only the container host
 
+          - image: quay.io/benz0li/ghc-musl:9.8.4
+            os: ubuntu-latest
+            static: true
```

Comment:
This is only x64 right? We'd need to have a similar one for ARM as well.

Reply:
Pandoc uses [Cirrus CI](https://cirrus-ci.org/) for their builds https://github.com/jgm/pandoc/blob/b57d3f9b4d9afbe05818c5161e502a525f813e7e/.cirrus.yml

Having to maintain two different CI solutions is unideal but it looks to be our only option for now...

Reply:
Let's try that!